demo/angularjs/components/search-field/doc.md

```diff
@@ -0,0 +1,17 @@
+# Search field
+
+**Search field allow users to specify a word or a phrase to find relevant content.**
```

Comment:
FIrst remark, `Search field` must be plural. 
I would stress the nature of the component, it is an input composed of a text field and a button.

Reply:
I agree for the nature description but it is more like a simple textfield in most cases and a textfield with a suggestion popover for the global search. So I tend to avoid describing its nature.